nip47/event_handler_legacy_test.go

```diff
@@ -1,192 +1,88 @@
 package nip47
 
 import (
-	"context"
```

Comment:
can we rename this file to `event_handler_shared_wallet_pubkey_test`?

Reply:
Done!